DoSomething/gladiator
# Update default messages

#### What's this PR do?

* Updates the content of the default messages set when you create a contest. 
* Fixes prove it link bug that was linking to the wrong anchor tag. 
* lower cases the :reportback_noun: and the :reportback_verb: tokens so that they look correct in a sentence.

#### How should this be manually tested?

Create a new contest with these messages and send yourself some tests. The content should match what is outlined in [this issue]( #215  )

#### Any background context you want to provide?

I wasn't able to test this a thoroughly as I wanted to on my dev because I got locked out of logging into gladiator cause of some issues on the northstar end. Hopefully this will get wrapped up by the time this gets deployed to QA and we can test there.

Another note, updating these default messages **will not do anything** for contests that have already had messages set up. We [have an issue]( #206 ) with a proposed solution for handling updating the default messages for contests that already exists.

#### What are the relevant tickets?
Fixes #215
